File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.cs
using System;

public class Solution {
    public int solution(int[] num_list) {
        
        int even = 0;
        for(int i = 0; i < num_list.Length; i += 2)
            even += num_list[i];
        
        int odd = 0;
        for(int i = 1; i < num_list.Length; i += 2)
            odd += num_list[i];
        
        return Math.Max(even, odd);
    }
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    
    int even = 0;
    int odd = 0;
    
    for (int i = 0; i < num_list.size(); i += 2)
        odd += num_list[i];
        
    for (int i = 1; i < num_list.size(); i += 2)
        even += num_list[i];
    
    return even > odd ? even : odd;
}